draft/accessibility-team.rst

```diff
@@ -0,0 +1,158 @@
+
+- A vote of the Technical Board, or full consensus of the rest of the
+  Accessibility Team, if the team is considered too large, the person is not
+  making positive contributions, or any other sound reason.
+
+Responsibilities
+----------------
+
+The remit of the Accessibility Team shall include all user-facing components
+of projects maintained by the Django Software Foundation. This includes but is
+not limited to:
+
+- User-facing parts of Django, such as default HTML output in forms and such,
+  and the django admin and its default theme, HTML, and JavaScript components.
+
+- All websites and other software projects maintained by the DSF, such as the
+  Django website, including documentation, Django Snippets, Django People,
+  the issue tracker, etc.
+
+The responsibilities of the Accessibility Team are expected to change over
+time, and are to be decided by consensus of the team, with input from the
+Technical Board as required. To begin, several areas have been identified:
+
+- Deciding on any relevant accessibility guidelines to follow, such as WCAG,
+  and at which conformance level.
+
+- Implementing automated testing to catch issues, working with the ops
+  team as needed to integrate this into the CI process.
+
+- Coordinating regular manual accessibility audits on all relevant projects.
+
+- Coordinating the fixing of accessibility issues and the improvement of the
+  accessibility in general in Django and associated projects.
+
+- Writing and maintaining documentation relating to accessibility, such as
+  a statement of commitment to accessibility issues, and contribution
+  guidelines.
+
+- Reviewing accessibility fixes, improvements and other tickets that may affect
+  the accessibility of the project.
+
+To aid in reviewing, the Accessibility Team members shall be added to a team
+in the GitHub organization with read access to relevant repositories, so that
+they may be requested to review pull requests, at the discretion of the author,
+reviewer, or other party.
+
+Many of these duties can be undertaken by any contributor, not only by the
+Accessibility Team, however the Accessibility Team exists to coordinate this
+work and to step in where contributors are not availble and support those who
+lack the knowledge to do so themselves.
+
+Motivation
+==========
+
+Accessible websites are not technically difficult to create, but accessibility
+is often overlooked when developing features. Django is no exception to this,
+and given Django's commitment to diversity and inclusion, the lack of
+accessibility in many parts of Django falls short of an acceptable standard.
+
+Improving accessibility also has a tendency to improve the experience of other
+users. This is called the
+`curb cut effect <https://alexwlchan.net/2019/01/monki-gras-the-curb-cut-effect/>`_
+While this is mostly noted in the real world, the web also experiences this
+affect. For example, keyboard shortcuts are used to allow keyboard-only users
```

Comment:
```suggestion
effect. For example, keyboard shortcuts are used to allow keyboard-only users
```

Reply:
I'm particularly glad you caught this before embarrassing myself with such mistakes in my _native tongue_ in the deps repo 😛